Dockerfile

```diff
@@ -0,0 +1,55 @@
+# First, get the elixir dependencies within an elixir container
+FROM elixir:1.11.3 AS elixir-builder
```

Comment:
Per @paulswartz's recommendation, we're standardizing on [`hexpm/elixir`](https://hub.docker.com/r/hexpm/elixir) now, because it pins both the Elixir and Erlang versions, so that we don't accidentally get a new Erlang version. Also you should use a `debian-buster` image since that's the final runtime you're targeting below.

Reply:
Also unless you're dependent on something that's only in Debian, Alpine is a lighter-weight container operating system.